Makefile

```diff
@@ -49,6 +49,10 @@ lint: virtualenv_test
 	$(FLAKE8) cunao.py
 
 test: virtualenv_test isort-check lint
+	( \
+	source .venv_test/bin/activate; \
+	pytest test/; \
+	)
```

Comment:
You should replace this for: `$(PYTEST) tests/`
Define `PYTEST` in the Makefile like `FLAKE8` for example